bitset.go

```diff
@@ -615,6 +630,12 @@ func (b *BitSet) Equal(c *BitSet) bool {
 		return true
 	}
 	wn := b.wordCount()
+	// bounds check elimination
+	if wn <= 0 {
+		return true
+	}
+	_ = b.set[wn-1]
```

Comment:
Can we fuse those in one line?

```
_, _ = b.set[wn-1], c.set[wn-1]
```

Reply:
I haven't checked the output of check_bce, but yeah, should be fine.